pkg/blockatlas/tx.go

```diff
@@ -66,7 +66,7 @@ type (
 	Block struct {
 		Number int64  `json:"number"`
 		ID     string `json:"id,omitempty"`
-		Txs    []Tx   `json:"txs"`
+		Txs    TxPage `json:"txs"`
```

Comment:
```suggestion
		Txs    []Tx `json:"txs"`
```

Reply:
Sorted via https://codebeautify.org/sort-text-lines, but this change let me fix that